scripts/test

```diff
@@ -1,20 +1,29 @@
-#!/bin/sh
+#!/bin/bash
 
-set -e
+declare -A test_results
+
+DOCKER_TLS_CERTDIR=""
+dockerd -H unix:///tmp/install-docker.sock & &>/dev/null
+
+export DOCKER_HOST="unix:///tmp/install-docker.sock"
+
+while true; do
+  docker version && break
+  sleep 5
+done
 
 echo "==> Testing install-docker scripts..."
 
-# https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
-SEMVER_REGEX='^(\d*)\.(\d*)\.(\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'
+cd pkg
 
-cd $(dirname $0)/../pkg
```

Comment:
Is not better to use relative path within repo?? It should work in a dapper or not dapper env